client/src/login/utils/tokenUtils.ts

```diff
@@ -0,0 +1,20 @@
+export const exchangeToken = (token: string): void => {
+    //Server token exchange
+};
+
+export const setToken = (token: string): void => {
+    window.localStorage.setItem('token', token);
+};
+
+export const decodeToken = (token: string): string => {
+    const [, payload] = token.split('.');
+    return JSON.parse(atob(payload));
+};
+
+// parse JWT from the URL hash
+export const getToken = (): string => {
```

Comment:
I know mocking the `window` object is a pain but these utils could use some tests too

Reply:
I havent added tests for these as the code isnt being used and will most likely change when the new auth implementation is in.